opencti-platform/opencti-front/src/private/components/locations/positions/PositionDetails.tsx

```diff
@@ -0,0 +1,238 @@
+    ) {
+        stixCoreRelationships(
+            elementId: $elementId
+            relationship_type: $relationship_type
+            confidences: $confidences
+            orderBy: $orderBy
+            orderMode: $orderMode
+            first: $count
+            after: $cursor
+        ) @connection(key: "Pagination_stixCoreRelationships") {
+            edges {
+                node {
+                    id
+                    entity_type
+                    parent_types
+                    relationship_type
+                    confidence
+                    start_time
+                    stop_time
+                    description
+                    is_inferred
+                    created_at
+                    to {
+                      ... on Position {
+                          name
+                          description
+                      }
+                      ... on City {
+                          id
+                          name
+                          description
+                          entity_type
+                      }
+                      ... on AdministrativeArea {
+                          id
+                          name
+                          description
+                          entity_type
+                      }
+                      ... on Country {
+                          id
+                          name
+                          description
+                          entity_type
+                      }
+                      ... on Region {
+                          id
+                          name
+                          description
+                          entity_type
+                      }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  paper: {
+    height: '100%',
+    minHeight: '100%',
+    margin: '10px 0 0 0',
+    padding: '15px',
+    borderRadius: 6,
+  },
+  chip: {
+    fontSize: 12,
+    lineHeight: '12px',
+    backgroundColor: theme.palette.background.accent,
+    borderRadius: 5,
+    color: theme.palette.text?.primary,
+    textTransform: 'uppercase',
+    margin: '0 5px 5px 0',
+  },
+}));
+
+interface PositionDetailsProps {
+  position: Position_position$data
+}
+
+const PositionDetails: FunctionComponent<PositionDetailsProps> = ({ position }) => {
+  const { t } = useFormatter();
+  const classes = useStyles();
+
+  return (
+    <QueryRenderer
+      query={
+        positionDetailsLocationRelationshipsLinesQuery
+      }
+      variables={{
+        count: 20,
+        elementId: [position.id],
+        relationship_type: 'located-at',
+      }}
+      render={({ props }: { props: PositionDetailsLocationRelationshipsLinesQueryLinesPaginationQuery$data }) => {
+        const targets = props?.stixCoreRelationships?.edges;
+        const cities = targets?.filter((n) => n?.node.to?.entity_type === 'City').map((n) => n?.node.to?.name);
```

Comment:
If we have a relationship, the node, to, to.id and to.entity_type property should not be null, what do you think ?